content/cloud/sqlite-storage.md

```diff
@@ -0,0 +1,123 @@
+title = "NoOps SQL Database"
+template = "cloud_main"
+date = "2023-07-26T16:00:00Z"
+enable_shortcodes = false
+[extra]
+url = "https://github.com/fermyon/developer/blob/main/content/cloud/noops-sql-database.md"
+
+---
+
+- [Prerequisites](#prerequisites)
+- [Accessing Private Beta](#accessing-private-beta)
+- [Limitations For NoOps SQL Database In Fermyon Cloud](#limitations-for-noops-sql-database-in-fermyon-cloud)
+- [How To](#how-to)
+- [Create a Table In Your NoOps SQL Database](#create-a-table-in-your-noops-sql-database)
+- [Manage The NoOps SQL Database Lifecycle](#manage-the-noops-sql-database-lifecycle)
+- [Next Steps](#next-steps)
+
+With NoOps SQL Database using SQLite support in Fermyon Cloud, you can persist relational data generated by your Spin application in between application invocations without having to worry about database management. Fermyon Cloud will provision and manage the database on your behalf. To learn more about the Spin SQLite SDK, please visit the [API guide](../spin/sqlite-api-guide.md).
+
+## Accessing Private Beta
+
+NoOps SQL Database in Fermyon Cloud is currently in private beta. To request access to the private beta, please fill out this short [sign-up form](https://fibsu0jcu2g.typeform.com/to/Brv12FI0#hubspot_utk=xxxxx&hubspot_page_name=xxxxx&hubspot_page_url=xxxxx).
+ 
+> Please note that the private beta is limited in space and all requests cannot be guaranteed. 
+
+## Service Limitations and Quotas For NoOps SQL Database in Fermyon Cloud
+
+*Service Limitations*
+* You must delete your Spin application before you can delete your NoOps SQL Database
```

Comment:
Sorry this is being nitty but why are we retaining "NoOps" adjective the whole time?

Reply:
No apologies needed! Nitty is good. 

We're retaining the adjective b/c its part of the feature name. We really want to drive home the point that NoOps is needed for this feature. I hear you though, its a long time which makes one start to crave TLAs again :-P

Reply:
@macolso I hear you get CEO brownie points if you call it "Noopsicle"